src/TweetPublisher/TweetPublisher.php

```diff
@@ -54,4 +70,12 @@ private function excludeAlreadyPublishedTweets(array $allPostTweets, array $allP
     {
         return array_diff_assoc($allPostTweets, $allPublishedTweets);
     }
+
+    /**
+     * @param string[] $tweetsToPublish
+     */
+    private function pickTweetCandidate(array $tweetsToPublish): string
```

Comment:
Trocha statiky z mé praxe vede více statiky.
Viz https://blog.codinghorror.com/the-broken-window-theory/

Preferuju tedy non-static first a vytvářet statiku, když je potřeba a nejde to jinak, např. v entitách.